source/mongodb-handshake/handshake.rst

```diff
@@ -224,20 +224,57 @@ Example::
         - "Oracle JVM EE 9.1.1"
 
 
-Limitations
+Supporting Wrapping Libraries
```

Comment:
`===...` below needs to be extended for a longer title.

Reply:
done